content/id/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Batas Rentang (Limit Range)
+content_template: templates/concept
+weight: 10
+---
+
+<!-- overview -->
+
+Secara bawaan, Container berjalan dengan [sumber daya komputasi](id/docs/user-guide/compute-resources) tanpa batas pada klaster Kubernetes.
+Dengan kuota sumber daya, administrator klaster dapat membatasi konsumsi dan pembuatan sumber daya berbasis {{<glossary_tooltip text="namespace" term_id="namespace">}}.
```

Comment:
ResourceQuota is a Kubernetes api object, need to add here as well

Namespace should start with uppercase
```suggestion
Dengan ResourceQuota (kuota sumber daya), administrator klaster dapat membatasi konsumsi dan pembuatan sumber daya berbasis {{<glossary_tooltip text="Namespace" term_id="namespace">}}.
```

Reply:
Thanks for the correction!